### PR TITLE
NAS-102924: Review update 1

### PR DESCRIPTION
**Pull request by Mrt134**
HTML build test: no issues.

### CODE REVIEW

File: userguide/jails.rst [hunk 1]
```diff
@@ -149,8 +149,8 @@ specified RELEASE directories as nullfs mounts over the jail
 directories. Basejails are not linked to the original RELEASE when
 upgraded.
 
-Choose the version of FreeBSD to install for this jail. The FreeBSD
-version cannot be newer than the FreeBSD version used by %brand%.
+Jails can run FreeBSD versions up to the same version as the host
+%brand% system. Newer releases are not shown.
```
Comment (a-stjohn) [inline userguide/jails.rst]:
```suggestion
%brand% system.
```

File: userguide/jails.rst [hunk 2]
```diff
@@ -255,8 +255,8 @@ a new jail.
    | Name                      | string            | Required. Jail names can only contain alphanumeric characters (:literal:`Aa-Zz`, :literal:`123`),       |
    |                           |                   | dashes (:literal:`-`), underscores (:literal:`_`), and periods (:literal:`.`).                          |
    +---------------------------+-------------------+---------------------------------------------------------------------------------------------------------+
-   | Release                   | drop-down menu    | Required. Choose the version of FreeBSD to download and install for the jail. The FreeBSD version       |
-   |                           |                   | cannot be newer than the FreeBSD version used by %brand%.                                               |
+   | Release                   | drop-down menu    | Required. Jails can run FreeBSD versions up to the same version as the host %brand% system.             |
+   |                           |                   | Newer releases are not shown.                                                                           |
```
Comment (a-stjohn) [inline userguide/jails.rst]:
```suggestion
   |                           |                   |                                                                                                         |
```

File: userguide/jails.rst (hunk 1 above)
Comment (Mrt134) [reply userguide/jails.rst]: That sentence was recommended to be added by Warren, so I'm hesitant to remove it...

Comment (a-stjohn) [reply userguide/jails.rst]: Fair enough. Just seems like if it's not shown, then there's no need to mention that?

**Pull request merged**